src/Symfony/Component/Console/Tests/Descriptor/AbstractDescriptorTest.php

```diff
@@ -102,6 +103,14 @@ protected function assertDescription($expectedDescription, $describedObject, arr
     {
         $output = new BufferedOutput(BufferedOutput::VERBOSITY_NORMAL, true);
         $this->getDescriptor()->describe($output, $describedObject, $options + array('raw_output' => true));
-        $this->assertEquals(trim($expectedDescription), trim(str_replace(PHP_EOL, "\n", $output->fetch())));
+
+        try {
+            $actual = trim(str_replace(PHP_EOL, "\n", $output->fetch()));
+            $this->assertEquals(trim($expectedDescription), $actual);
+        } catch (ExpectationFailedException $exception) {
+//            \var_dump($actual);
```

Comment:
To be removed I guess

Reply:
OMG... Sorry about... You're right it shouldn't be there :)

Reply:
Don't worry it happens :slightly_smiling_face: